web/components/elements/EventHookCheckbox.tsx

```diff
@@ -0,0 +1,26 @@
+import { FunctionComponent } from "preact";
+import { JSXInternal } from "preact/src/jsx";
+
+export const EventHookCheckbox: FunctionComponent<{
+    enabledHooks?: string[],
+    onChange: JSXInternal.GenericEventHandler<HTMLInputElement>,
+    eventName: string,
+    parentEvent?: string,
+}> = ({enabledHooks, onChange, eventName, parentEvent, children}) => {
+    if (!enabledHooks) {
+        throw Error(`Invalid configuration for checkbox ${eventName}`);
```

Comment:
After 5 years of coding with React, this is the first component I see which intentionally `throw`s.
Are you sure you don't want to display something?

Reply:
Tbh, I think type checking is enough. I'll remove the error.